include/amici/model.h

```diff
@@ -691,6 +691,19 @@ namespace amici {
          */
         void setTimepoints(std::vector<realtype> const& ts);
 
+        
+        /**
+         * @brief gets flags indicating whether states should be treated as non-negative
+         * @return vector of flags
+         */
+        std::vector<bool> getQPositiveX() const;
```

Comment:
What does the Q stand for?

Reply:
😅 thats how it was called in d2d

Reply:
:thinking: Maybe we can go for something like `nonNegativeStates` `nonNegativeStateIndices`?